src/Hopper/Internal/LoweredCore/ClosureConvertedANF.hs

```diff
@@ -42,11 +42,9 @@ import Hopper.Internal.Core.Literal
 import GHC.Generics
 import qualified  Data.Vector as V
 import Hopper.Internal.Type.Relevance(Relevance)
-import Hopper.Internal.Runtime.Heap(TransitiveLookup(..),heapLookup)
+import Hopper.Internal.Runtime.Heap
```

Comment:
for unqualified imports, lets over time try to have those have explicit export lists, but this is fine for now